src/pages/blog/index.html

```diff
@@ -0,0 +1,37 @@
+<html>
+  <head>
+    <title>Greenwood - ${globalThis.page.title}</title>
+    <meta property="og:title" content="Greenwood - ${globalThis.page.title}" />
+    <script
+      type="module"
+      src="../../components/blog-posts-list/blog-posts-list.js"
+      data-gwd-opt="static"
+    ></script>
+    <style>
+      .container {
+        text-align: center;
+        margin: var(--size-2);
+      }
+
+      .heading {
```

Comment:
Good call out from @aholtzman 
From the designs, we need to make this text a _lot_ bigger and with the "pyramid" style of the news and text (e.g. the line break)
![Screenshot 2024-08-28 at 7 37 19 PM](https://github.com/user-attachments/assets/5ff42094-67ce-4688-88da-360cf1f5ef2b)